pkg/cli/admin/top/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+# This file just uses aliases defined in OWNERS_ALIASES.
+
+reviewers:
+  - gmeghnag
+  - ardaguclu
```

Comment:
It is better to add approvers and reviewers from which team will maintain this. I'd prefer not to be in here.